mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -20,12 +22,13 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 	METHOD method_8089 toJson ()Lcom/google/gson/JsonElement;
 	METHOD method_8091 ofItems ([Lnet/minecraft/class_1935;)Lnet/minecraft/class_1856;
 	METHOD method_8092 ofEntries (Ljava/util/stream/Stream;)Lnet/minecraft/class_1856;
-	METHOD method_8096 createStackArray ()V
+	METHOD method_8096 cacheMatchingStacks ()V
 	METHOD method_8100 getIds ()Lit/unimi/dsi/fastutil/ints/IntList;
 	METHOD method_8101 ofStacks ([Lnet/minecraft/class_1799;)Lnet/minecraft/class_1856;
 	METHOD method_8102 fromJson (Lcom/google/gson/JsonElement;)Lnet/minecraft/class_1856;
-		ARG 0 element
+		ARG 0 jsonElement
 	METHOD method_8103 isEmpty ()Z
-	METHOD method_8105 getStackArray ()[Lnet/minecraft/class_1799;
+	METHOD method_8105 getMatchingStacks ()[Lnet/minecraft/class_1799;
```

Comment:
Consider appending `client` to the name of the method, could avoid stupid crashes.

Reply:
What do you mean? You want it to be called `getMatchingStacksClient()`?

Reply:
That's what I would suggest. Just like HungerManager#setSaturationLevelClient. Just something to signal that you cannot use it on the server.

Reply:
right